tests/unittest/benchmark_runner/common/template_operations/golden_files/test_ci_windows_vm_ODF_PVC_True/windows_dv.yaml

```diff
@@ -16,4 +16,4 @@ spec:
       requests:
         storage: 70Gi
     volumeMode: Block
-    storageClassName: ocs-storagecluster-ceph-rbd-virtualization
\ No newline at end of file
+    storageClassName: ocs-storagecluster-ceph-rbd-virtualization
```

Comment:
Seems you removing newline in all windows dv file, am I right ?

Reply:
Yep, that's fixed up now so there's the one trailing newline.